service/frontend/workflowHandler.go

```diff
@@ -3225,3 +3267,201 @@ func (wh *WorkflowHandler) getArchivedHistory(
 		Archived:      common.BoolPtr(true),
 	}, nil
 }
+
+func (wh *WorkflowHandler) convertIndexedKeys(keys map[string]interface{}) map[string]gen.IndexedValueType {
```

Comment:
I'm not sure what this is converting? It seems to return the same key/value pair back.

Reply:
Go requires such type cast from interface{} to real type. I renamed it `convertIndexedKeysToThrift`